src/main/scala/se/nimsa/sbx/metadata/MetaDataDAO.scala

```diff
@@ -44,8 +44,8 @@ class MetaDataDAO(val dbConf: DatabaseConfig[JdbcProfile])(implicit ec: Executio
 
   class PatientsTable(tag: Tag) extends Table[Patient](tag, PatientsTable.name) {
     def id = column[Long]("id", O.PrimaryKey, O.AutoInc)
-    def patientName = column[String](DicomProperty.PatientName.name, O.Length(512))
```

Comment:
Patient names can be up to 64*5+4=324 characters according to the DICOM standard. If we are to support MySQL and utf-8, this is about the best we can do. If we write a DICOM conformance statement, this limitation should be detailed there.